content/en/core/releases/3.14.0.md

```diff
@@ -0,0 +1,143 @@
+
+Sync in progress:
+![](../images/3.14.0-5207-current-sync.png)
+
+Sync complete:
+![](../images/3.14.0-5207-sync-done.png)
+
+Sync failed (with retry option):
+![](../images/3.14.0-5207-sync-failed.png)
+
+[#5207](https://github.com/medic/cht-core/issues/5207): Show dialog when user clicks "Sync Now" in hamburger menu
+
+#### Updated icon for the Create Report button
+
+The icon for the Create Report button has been updated so that its purpose is more clear.
+
+Old Icon | New Icon
+:-------------------------:|:-------------------------:
+![](../images/3.14.0-6403-old.png)  |  ![](../images/3.14.0-6403-new.png)
+
+[#6403](https://github.com/medic/cht-core/issues/6403): Change the Create Report button icon to better indicate what it does
+
+#### Display all dates in Bikram Sambat format when Nepali locale selected
+
+For years, the CHT has supported recording the answers to `date` questions in forms using the [Bikram Sambat](https://en.wikipedia.org/wiki/Vikram_Samvat) calendar format when the user has selected the Nepali locale. However, the date values displayed throughout the app would still be shown in the [Gregorian](https://en.wikipedia.org/wiki/Gregorian_calendar) calendar format.
+
+Now the application has been updated to also use the Bikram Sambat format when displaying dates in places such as the contact summary.
+
+![](../images/3.14.0-7294-contact-summary.png)
+
+Additionally, a new [`to-bikram-sambat` xPath function]({{< ref "apps/reference/forms/app#to-bikram-sambat" >}}) has been added that converts a `date` to a `string` containing the value of the date in the Bikram Sambat format.  
+
+[#7294](https://github.com/medic/cht-core/issues/7294): Use Bikram Sambat dates throughout the webapp when Nepali locale selected
+
+#### Properly display tab labels on small screens
+
+Previously, the labels for the main tabs in the CHT app (Messages, Tasks, People, etc) were not displayed on devices with very small screens (leading to some confusion around the different tabs and their purposes).
+
+We have updated the tab labels to always display (even on small screens). Additional the size of the label text can scale so that the text will display properly even when the tabs are small.
+
+[#7409](https://github.com/medic/cht-core/issues/7409): Display tab's label in small screens
+
+
+## Highlights
+
+### Improved server-side purging
+
+We have made several improvements to our [server-side purging]({{< ref "apps/guides/performance/purging#server-side" >}}) functionality allowing it to perform better at scale.
+
+- Enhanced logging during purging via a new [`purgelog` document]({{< ref "apps/guides/performance/purging#purged-documents-server-side" >}}) saved in the `medic-sentinel` database after every purge.
+- If a contact has more than 20.000 records, it will be skipped (none of its records will be purged). 
```

Comment:
```suggestion
- If a contact has more than 20,000 records, it will be skipped and none of its records will be purged.
```

US English uses comma separators for thousands IIRC.

Reply:
Hah! I originally copied some of this wording from a comment Diana had left on on the issue. Forgot to proofread for locale specific number formatting (ironic given the issue in question...).